[Term Entry] Spyware (#2767)
Codecademy/docs: content/cybersecurity/concepts/malware/terms/spyware/spyware.md

---
Title: 'Spyware' 

> Description: 'Spyware is malicious software that enters a person's computer, takes their data, and sends it to other third party apps or companies without the user's permission.' 

Reviewer: The current description is giving an error in YAML due to the apostrophes being picked up as closing single quotation marks. I provided a possible edit for this below:
```suggestion
Description: 'Spyware is malicious software designed to gather personal data from a device in order to send it to other third-party apps or companies without consent from the user.' 
```

Subjects: 
  - 'Cybersecurity'
  - 'Computer Science'
Tags:
  - 'Cybersecurity'
  - 'Computer Science'
CatalogContent: 

>   - 'paths/cyber-resilience-risk-management'

Reviewer: There should be a free course listed 
```suggestion
  - 'introduction-to-cybersecurity'
  - 'paths/cyber-resilience-risk-management'
```

  - 'paths/fundamentals-of-cybersecurity'
---

>  **Spyware** is malicious computer software that takes a user's personal data and sends it to other companies and apps without the person's consent or knowledge.

Reviewer: ```suggestion
 **Spyware** is malicious computer software designed to gather personal data from a device in order to send it to other third-party apps or companies without the consent or knowledge of the user.
```

> ## Why is Spyware harmful?

Reviewer: Title case headings (AP Style)
```suggestion
## Why Is Spyware Harmful?
```

Since Spyware steals a person's information and sends it others, it can put the person in all sorts of situations whether it's receiving bills for unintended purchases, private information being leaked to the public, or prevent you from signing in to the user's Apple or Google Play account to download music and apps. In other words, spyware ruins lives.

> ## How does Spyware infect devices?

Reviewer: ```suggestion
## 	How Does Spyware Infect Devices?
```

Spyware infects a person's devices in the following ways

- Clicking on a suspicious looking prompt or pop-up window
- Opening email attachments from an unknown person
- Downloading software and other forms of media from an unsecured website
  

> ## Signs of a Spyware-infected device

Reviewer: ```suggestion
## 	Signs of a Spyware-Infected Device
```

- Slow performance and unexpected crashes  
- Lack of hard drive and/or cloud storage
- Icons to programs that are downloaded without user permission

> ## Spyware prevention tips

Reviewer: ```suggestion
## Spyware Prevention Tips
```

To prevent spyware from infecting a device and ruining lives, consider using the following strategies:

> - Avoid clicking on popup ads

Reviewer: ```suggestion
- Avoid clicking on pop-up ads
```

---
Title: 'Spyware'
Description: 'Spyware is malicious software designed to gather personal data from a device in order to send it to other third-party apps or companies without consent from the user.'
Subjects:
  - 'Cybersecurity'
  - 'Computer Science'
Tags:
  - 'Cybersecurity'
  - 'Computer Science'
CatalogContent:
  - 'introduction-to-cybersecurity'
  - 'paths/cyber-resilience-risk-management'
  - 'paths/fundamentals-of-cybersecurity'
---

**Spyware** is malicious computer software designed to gather personal data from a device in order to send it to other third-party apps or companies without the consent or knowledge of the user.

## Why Is Spyware Harmful?

> Since Spyware steals a person's information and sends it others, it can put the person in all sorts of situations whether it's receiving bills for unintended purchases, private information being leaked to the public, or prevent you from signing in to the user's Apple or Google Play account to download music and apps. In other words, spyware ruins lives.

Reviewer 2: ```suggestion
Since Spyware steals a person's information and sends it others, it can put the person in all sorts of situations; whether it's receiving bills for unintended purchases, private information being leaked to the public, or preventing signing in to the user's Apple or Google Play account to download music and apps. In other words, spyware ruins lives.
```

## How Does Spyware Infect Devices?

> Spyware infects a person's devices in the following ways

Reviewer 2: ```suggestion
Spyware infects a person's devices in the following ways:
```

- Clicking on a suspicious looking prompt or pop-up window
- Opening email attachments from an unknown person
- Downloading software and other forms of media from an unsecured website

## Signs of a Spyware-Infected Device

- Slow performance and unexpected crashes
- Lack of hard drive and/or cloud storage
- Icons to programs that are downloaded without user permission

## Spyware Prevention Tips

To prevent spyware from infecting a device and ruining lives, consider using the following strategies:

> - Avoid clicking on pop-up ads
> - Click on the trash icon when receiving emails that have recurring grammatical errors and containing messages where someone is asked to send personal information such as a bank account number or home address to solve a problem
> - Purchase reputable anti-virus software such as [Norton](https://us.norton.com/) [McAfee](https://www.mcafee.com/)

Reviewer 2: ```suggestion
- Avoid clicking on pop-up ads.
- Click on the trash icon when receiving emails that have recurring grammatical errors and containing messages where someone is asked to send personal information such as a bank account number or home address to solve a problem.
- Purchase reputable anti-virus software such as [Norton](https://us.norton.com/) or [McAfee](https://www.mcafee.com/).
```
